### DISCUSSION TITLE
Image Preview failed

### DISCUSSION THREAD

**Discussion by bubbawilliams** [skills/.github#519] [Communicate using Markdown]
![Speeding Train](https://www.istockphoto.com/photo/speeding-subway-train-in-brooklyn-gm1291459949-386569853)

doesn't show in Preview.

I've tried it with and without the space between "]" and "(".

Any Help Appreciated,

BW

**Comment by sinsukehlab** [comment]
I'm afraid your link is not an image.

**Comment by aflacake** [comment]
If you don't have an online link image, you should use the image in the steps:
```md
![Image of Yaktocat](https://octodex.github.com/images/yaktocat.png)
```
